lib/fhir_models/bootstrap/definitions.rb

```diff
@@ -211,6 +211,32 @@ def self.get_codes(uri)
       codes
     end
 
+    # Get the "display" (human-readable title) for a given code in a code system (uri)
+    # If one can't be found, return nil
+    def self.get_display(uri, code)
+      return nil if uri.nil? || code.nil?
+      load_expansions
+      valuesets = @@expansions.select { |x| !x['compose']['include'].select {|i| i['system'] == uri }.empty? }
+      valuesets += @@valuesets.select { |x| x['url'] == uri }
+      c = nil
+      valuesets.each do |valueset|
```

Comment:
Good points @wesrich... we should fix that. I guess it'll be another pull request since I've hastily merged it.